lib/datasource/remote/firebase/firebase_remote_config.dart

```diff
@@ -71,7 +71,7 @@ class _FirebaseRemoteConfigService {
 
   bool get featureFlagImportAccountEnabled => _remoteConfig.getBool(_featureFlagImportAccount);
 
-  String get hyphaEndPoint => testnetMode ? _testnet_hyphaEndPointUrl : _remoteConfig.getString(_hyphaEndPointUrl);
```

Comment:
Guys... This is not a proper fix. 

`_remoteConfig.getString(_hyphaEndPointUrl);` you can see that we are trying to fetch using the `_hyphaEndPointUrl` instead of using the `_hyphaEndPointKey`

the proper fix is to change it to _remoteConfig.getString(_hyphaEndPointKey);